ceph.md

```diff
@@ -233,12 +218,16 @@ spec:
         store_description = "RBD backend"
         rbd_store_pool = images
         rbd_store_user = openstack
-        [image_import_opts]
-        image_import_plugins = ['image_conversion']
-        [image_conversion]
-        output_format = raw
```

Comment:
This is now handled by the operator if necessary.

https://github.com/openstack-k8s-operators/glance-operator/blob/89a26154405ffd020ac05c761fdc6ebf4c2d3775/templates/common/config/00-config.conf#L118-L125